src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -413,24 +414,121 @@ private static string EmitIndexOfAnyValues(char[] asciiChars, Dictionary<string,
                 _ => $"s_ascii_{hexBitmap.TrimStart('0')}"
```

Comment:
Given how common such character classes are, would it make sense to add special-cased names here for them?
```c#
"FFFFFFFFFFFF00FCFFFFFFFFFFFFFFFF" => "s_ascii_anyExceptDigits",
"FFFFFFFFFFFFFFFF010000F8010000F8" => "s_ascii_anyExceptLettersAndDigits",
...
```
?

Reply:
Good idea.  I expanded this list as well as the list of helper method names.